v0/destinations/canny/data/identifyConfig.json

```diff
@@ -34,7 +34,12 @@
   },
   {
     "destKey": "avatarURL",
-    "sourceKeys": ["context.traits.avatarURL", "traits.avatarURL"],
+    "sourceKeys": [
+      "context.traits.avatar",
```

Comment:
We give precedence to the top level traits over the context.traits. Let's fix that

Reply:
Updated.